### DISCUSSION TITLE
Add profile README for Robolectric with .github repo

### DISCUSSION THREAD

**Discussion by utzcoz** [robolectric/robolectric#7053] [Ideas]
The profile `README.md` can be used a entry for Robolectric org, and introduce useful links for users and developers when they view https://github.com/robolectric/. There is an example of flutter's profile `README.md`: https://github.com/flutter/.

**Comment by MGaetan89** [comment]
Digging up old topics here, but it seems like a good idea!
We'll need a `.github` repository created in the @robolectric organisation that contains a `README.md` file: https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile
We don't need to put too much information in it, but I suggest:
- Project description
- Link to the website
- Link to the main repositories, with a small description:
  - robolectric/robolectric
  - robolectric/robolectric-bazel
  - robolectric/robolectric.github.io
- Link to the contribution guide

-----

Also, it may be interesting to link the [robolectric.org](https://robolectric.org) domain to the @robolectric organisation: https://docs.github.com/en/organizations/managing-organization-settings/verifying-or-approving-a-domain-for-your-organization
This will add a `Verified` badge on the organisation page (see @flutter for an example).

-----

~Small detail: maybe the project link should be changed to use https:~ done ✅
<img width="403" alt="Screenshot 2024-11-19 at 16 48 47" src="https://github.com/user-attachments/assets/d3372461-06f7-47e4-8f88-522a3c7ecc05">
